packages/test-base/src/nativeDarwinTest/kotlin/io/realm/kotlin/test/darwin/MemoryTests.kt

```diff
@@ -57,63 +55,6 @@ class MemoryTests {
         deleteTempDir(tmpDir)
     }
 
-    // TODO Only run on macOS, filter using https://developer.apple.com/documentation/foundation/nsprocessinfo/3608556-iosapponmac when upgrading to XCode 12
```

Comment:
Why are we removing this test? The other test is verifying that garbage collecting is actually cause a `realm_release` but that is just a subset of ensure that we actually release all resource when not referencing a realm anymore.